landoui/auth.py

```diff
@@ -0,0 +1,79 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+import os
+
+"""Class that governs all authentication with open id connect."""
+from flask_pyoidc.flask_pyoidc import OIDCAuthentication
+
+
+class nullOpenIDConnect(object):
```

Comment:
Class names should start with a capital letter.

You can drop the inheritance from `object` in Python 3.  (It's a Python 2 idiom.)

The class definitions in this module should look like this:

```python
class NullOpenIDConnect:

    blah
```